tests/JobCreatorTest.php

```diff
@@ -1949,4 +1949,46 @@ public function provideComposerInstall(): array
             ],
         ];
     }
+
+    public function testDuplicateJobsRemoved(): void
+    {
+        if (!function_exists('yaml_parse')) {
+            $this->markTestSkipped('yaml extension is not installed');
+        }
+        $yml = implode("\n", [
+            $this->getGenericYml(),
+            <<<EOT
+            github_repository: 'myaccount/silverstripe-framework'
+            github_my_ref: '5'
+            parent_branch: ''
+            extra_jobs:
+              - php: 8.2
+                phpunit: true
+                phpunit_suite: fish
+              - php: 8.3
+                phpunit: true
+                phpunit_suite: fish
+              - php: 8.3
+                phpunit: true
+                phpunit_suite: fish
+              - php: 8.3
+                endtoend: true
```

Comment:
None of these are duplicated. Can you please add something that's very obviously duplicated into this test?
I suspect these are duplicates of what's in `getGenericYml` but that's not clear by looking at the test.

Reply:
Nevermind, I'm just blind I guess lol. One of those is clearly a dupe.